shared/js/background/https.es6.js

```diff
@@ -226,7 +215,6 @@ class HTTPS {
         const testHosts = [
             '1337x.to',
             'submit.pandora.com',
-            '*.api.roblox.com',
```

Comment:
I couldn't actually get the tests to work :man_shrugging: 

I'm hoping to fix them soon, so at least I wanted to keep the test cases a bit more up-to-date.